services-camel-app/src/main/webapp/WEB-INF/solr-indexing-context.xml

```diff
@@ -284,6 +286,9 @@
         <property name="memberOrderService" ref="memberOrderService" />
     </bean>
 
+    <bean id="setViewBehaviorFilter" class="edu.unc.lib.boxc.indexing.solr.filter.SetViewBehaviorFilter">
```

Comment:
Since a new solr indexing action was created, we'll need to create an action and a pipeline for it, and map what happens when its received down here:
https://github.com/UNC-Libraries/box-c/blob/ab4301ab9de42979c1376ab4b5763e7d3672fe75/services-camel-app/src/main/webapp/WEB-INF/solr-indexing-context.xml#L456
The pipeline probably would have setRecordDatesFilter and setViewBehaviorFilter in it (the first one updates modification dates), and the action would be probably use the `edu.unc.lib.boxc.indexing.solr.action.UpdateObjectAction` class since we only need to update the work record itself

Reply:
I think I've added this now!